internal/buildscripts/packaging/tests/installer_test.py

```diff
@@ -81,6 +81,21 @@ def test_installer(distro, version, memory_option):
             else:
                 assert container.exec_run("systemctl status td-agent").exit_code != 0
 
+            # test support bundle script
+            assert container.exec_run("/etc/otel/collector/splunk-support-bundle.sh -t /tmp/splunk-support-bundle").exit_code == 0
```

Comment:
This will fail until we release the deb/rpm packages with this new file.  Probably best to check if the file exists first, i.e.:
```
if container.exec_run("test -f /etc/otel/collector/splunk-support-bundle.sh").exit_code == 0:
    assert container.exec_run("/etc/otel/collector/splunk-support-bundle.sh -t /tmp/splunk-support-bundle").exit_code == 0
```